### PR TITLE
docs: add `new` command reference

### PR DESCRIPTION
**Pull request by renancaraujo**
## Status

**READY**

## Description

docs: add `new` command reference

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: docs/docs/basics/middleware.md [hunk 1]
```diff
@@ -36,3 +36,11 @@ Handler middleware(Handler handler) {
   return handler.use(requestLogger());
 }
 ```
+
+## Creating Middleware
+
+To create a new middleware on a Dart Frog project, run the following command:
```
Comment (scarletteliza) [inline docs/docs/basics/middleware.md:42]:
```suggestion
To create new middleware on a Dart Frog project, run the following command:
```

File: docs/docs/basics/middleware.md [hunk 2]
```diff
@@ -36,3 +36,11 @@ Handler middleware(Handler handler) {
...
+
+```bash
+dart_frog new middleware "/hello" # Creates routes/hello/_middleware.dart
```
Comment (erickzanardo) [inline docs/docs/basics/middleware.md:45]:
```suggestion
# Creates routes/hello/_middleware.dart
dart_frog new middleware "/hello"
```

File: docs/docs/basics/routes.md [hunk 3]
```diff
@@ -19,6 +19,14 @@ Response onRequest(RequestContext context) {
 }
 ```
 
+## Creating Routes
+
+Managing routes in Dart Frog is essentially as simple as managing the file structure under `/routes`. To help on the creation of routes, Dart Frog provides a CLI command to generate a new route.
+
+```shell
+dart_frog new route "/hello" # will create routes/hello.dart
```
Comment (erickzanardo) [inline docs/docs/basics/routes.md:27]:
```suggestion
# will create routes/hello.dart
dart_frog new route "/hello"
```

**Comment by CodeDoctorDE** [comment]
Maybe a `dart_frog new main` command to create a custom main file with the methods run and init?

**Review by alestiago — APPROVED** [review]

**Review by scarletteliza — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -66,6 +66,18 @@ Create a production build which includes a `DockerFile` so that you can deploy a
...
 
+### Create New Routes and Middleware 🛣️
+
+To add new routes and middleware to your project, use the `dart_frog new` command.
+
+```sh
+# 🛣️ Create a new route "/hello/world"
+dart_frog new route "/hello/world"
+
+# 🛣️ Create a new middleware for the route "/hello/world"
```
Comment (CodeDoctorDE) [inline README.md:77]: Maybe a better icon?
:station: for middleware or :stop_sign:?

**Review by CodeDoctorDE — APPROVED** [review]
I'm not a contributor but for me, it looks good

**Pull request merged**